packages/package-json-lint/src/utilities/index.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs-extra');
+const callbackGlob = require('glob');
+const util = require('util');
+const path = require('path');
+const ignore = require('ignore');
+
+const IGNORE_FILE_PATH = path.join(process.cwd(), '.packagejsonlintignore');
+
+const glob = util.promisify(callbackGlob);
+
+const getIgnorer = async () => {
+  const ignoreFileContents = await fs.readFile(IGNORE_FILE_PATH, 'utf8');
+  return ignore().add(ignoreFileContents);
+};
+
+const getPathsForPackages = async () => {
+  const paths = await glob(path.join(process.cwd(), '**', 'package.json'), { ignore: [path.join('**', 'node_modules', '**')] });
```

Comment:
should we always assume a package at root and look for a lerna config to glob for mono repos? looking to avoid unnecessary globs.

Reply:
I was wondering if we would want to use this in a server as well.

Reply:
Updated this to cover all scenarios